solana-common/proto/instructions.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
```

Comment:
@enoldev I was discussing with Charles and Ed and they think we should start with only transaction. They think that getting instructions alone is not as useful because usually you also need to access other instruction within the same transaction which is not easily modelled now.

Let's discuss this Monday in greater details.